write_config.js

```diff
@@ -0,0 +1,17 @@
+const { RainbowConfig } = require('rainbow-bridge-utils')
```

Comment:
This file is listed in .gitignore. Did you mean to still push it?

Reply:
++